pkgs/servers/code-server/default.nix

```diff
@@ -1,12 +1,12 @@
 { lib, stdenv, fetchFromGitHub, buildGoModule, makeWrapper
 , cacert, moreutils, jq, git, rsync, pkg-config, yarn, python3
-, esbuild, nodejs_16, node-gyp, libsecret, xorg, ripgrep
+, esbuild, nodejs_18, node-gyp, libsecret, xorg, ripgrep
```

Comment:
why did you change to nodejs 18, code-server officially still only supports 16
https://github.com/coder/code-server/blob/2109d1cf6abdf17e620a926239d9e96a07face4d/.github/workflows/release.yaml#LL38C13-L38C13

Reply:
`nodejs_16` has been deprecated, so I'm just checking if this is working with NodeJS 18. I already made the change in openvscode-server at https://github.com/NixOS/nixpkgs/pull/232887